### PR TITLE
ci: add pr change log fill check

### PR DESCRIPTION
**Pull request by xrkffgg**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Other (about what?)

### 💡 Background and solution
增加 PR changelog  填写检测

未填写时，会评论提醒，CI 状态提醒
填写后，评论自动删除，CI 状态正常

See: https://github.com/xrkffgg/ant-design/pull/5 https://github.com/xrkffgg/ant-design/pull/7

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |       -    |
| 🇨🇳 Chinese |     -      |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: .github/workflows/pr-open-check.yml [hunk 1]
```diff
@@ -18,3 +18,23 @@ jobs:
...
+      - name: check fill
+        uses: actions-cool/pr-check-fill@v1.0.0
+        with:
+          filter-start: '|'
+          require-include: '🇺🇸 English, 🇨🇳 Chinese'
+          comment-body: |
+            - 🚨 Please fill changelog in the PR
+              - Write with a **developer-oriented perspective** and **narrative method**, without describing the details of the repair
+              - **Describing the problem and the impact on the developer**
+              - **describing the user-first site problem**, not your solution
```
Comment (afc163) [inline .github/workflows/pr-open-check.yml:34]: 给个 changelog 链接做例子。

File: .github/workflows/pr-open-check.yml [hunk 2]
```diff
@@ -18,3 +18,23 @@ jobs:
...
+
+            - 🚨 请填写 PR 中的 changelog
+              - 请用**面向开发者的角度**和**叙述方式撰写**，不描述修复细节
+              - **描述问题和对开发者的影响**
+              - **描述用户第一现场的问题**，而非你的解决方式
+          skip-title-start: 'docs'
```
Comment (afc163) [inline .github/workflows/pr-open-check.yml:40]: chore:
test:

这些也不用给。

File: .github/workflows/pr-open-check.yml (hunk 1 above)
Comment (xrkffgg) [reply .github/workflows/pr-open-check.yml:34]: https://github.com/ant-design/ant-design/pull/29781 这个例子如何？

Comment (afc163) [reply .github/workflows/pr-open-check.yml:34]: 可以是 https://ant.design/changelog-cn#4.9.0 和 https://ant.design/changelog#4.9.0

Comment (xrkffgg) [reply .github/workflows/pr-open-check.yml:34]: OK

**Review by afc163 — APPROVED** [review]

**Pull request merged**